deployment/run_server

```diff
@@ -40,7 +40,7 @@ pkill -9 Xvfb
 pkill -9 node
 pkill -9 electron
 
-xvfb-run --auto-servernum --server-args '-screen 0 640x480x24' /var/www/image-exporter/bin/orca.js serve $REQUEST_LIMIT --safe-mode --verbose $PLOTLYJS_ARG $ORCA_IGNORECERTERRORS_ARG $@ &
+xvfb-run --auto-servernum --server-args '-screen 0 1024x768x24' /var/www/image-exporter/bin/orca.js serve $REQUEST_LIMIT --safe-mode --verbose $PLOTLYJS_ARG $ORCA_IGNORECERTERRORS_ARG $@ &
```

Comment:
Is this really necessary? I thought this line: `createBrowserWindowOpts['enableLargerThanScreen'] = true` would make the result independent of the screen resolution :thinking:

Reply:
I was under the same impression, but apparently it doesn't. Should I remove that option, or let it be there?

Reply:
That is unfortunate... I wonder if that is fixed in recent versions of `electron` (see branch [`pr-bump-electron-6`](https://github.com/plotly/orca/tree/pr-bump-electron-6)). 

Anyway, changing the resolution of the screen seems like a short-term solution: what if a user needs a bigger browser size than 1024 by 768... We're essentially putting a limit on how big a page we can print. If so, it should be clearly stated somewhere. By the way, what do you mean when you say it "works" (ie. how/what are you testing exactly)?